freqtrade/freqai/base_models/BasePyTorchClassifier.py

```diff
@@ -149,3 +161,57 @@ def get_class_names(self) -> List[str]:
             )
 
         return self.class_names
+
+    def train(
```

Comment:
Is there any particular difference between this and BaseTorchRegressor i'm missing? 

The 2 methods seem to be identical (and both inherit from the same parent) - so i think they should be / remain in the parent class.

Reply:
Yes, the regressor also has a pipeline for the labels. Meanwhile the classifier does not.